src/components/views/rooms/EventTile.js

```diff
@@ -696,6 +704,9 @@ export default createReactClass({
             // joins/parts/etc
             avatarSize = 14;
             needsSenderProfile = false;
+        } else if (this.props.useIRCLayout) {
+            avatarSize = 14;
```

Comment:
This is generally true. However I don't know the code here well enough to change the order of these conditionals in a semantically equivelent way. Right now the css assumes no profile for  the `else if (isInfoMessage) {` case and I can't tell if I can swap the order of the `if (this.props.tileShape === "notif") {` and the `} else if (tileHandler === 'messages.RoomCreate' || isBubbleMessage) {` cases, which is necessary because the `tileHandler` case needs to come before the irclayout choice.

Reply:
Ok, I'm also not entirely sure tbh. I think this is definitely not a blocker, so good to go :+1: